python/taichi/lang/impl.py

```diff
@@ -300,7 +300,44 @@ def layout(func):
 
 
 def ti_print(*vars):
-    taichi_lang_core.create_print([Expr(var).ptr for var in vars])
+    def ent2content(var):
```

Comment:
What does `ent` mean? If it's `entry` let's use the full name. Not too much longer anyway.